vmm/src/vm.rs

```diff
@@ -538,6 +538,11 @@ impl Vm {
         let numa_nodes =
             Self::create_numa_nodes(config.lock().unwrap().numa.clone(), &memory_manager)?;
 
+        #[cfg(feature = "tdx")]
+        let force_iommu = false;
+        #[cfg(not(feature = "tdx"))]
+        let force_iommu = true;
+
```

Comment:
This should only be set if TDX is in use not if it is only compiled in.

Reply:
Done!